fixtures/features/api-elements/produces-header.sourcemap.json

```diff
@@ -91,6 +91,44 @@
                             ]
                           },
                           "content": "GET"
+                        },
+                        "headers": {
+                          "element": "httpHeaders",
+                          "meta": {},
+                          "attributes": {},
+                          "content": [
+                            {
+                              "element": "member",
+                              "meta": {
+                                "links": [
+                                  {
+                                    "element": "link",
+                                    "meta": {},
+                                    "attributes": {
+                                      "relation": "inferred",
+                                      "href": "http://docs.apiary.io/validations/swagger#produces-accept"
+                                    },
+                                    "content": []
```

Comment:
Why is this an array? The spec doesn't mention that this element has a content value.

https://github.com/refractproject/refract-spec/blob/master/refract-spec.md#link-element-element